eng/tox/tox.ini

```diff
@@ -129,7 +129,8 @@ changedir = {toxinidir}
 deps =
   {[tools]deps}
 commands = 
-    {envbindir}/python {toxinidir}/../../../eng/tox/create_wheel_and_install.py -d {distdir} -p {toxinidir}
+    {envbindir}/python {toxinidir}/../../../eng/tox/create_wheel_and_install.py -d {distdir} -p {toxinidir} \
+      -e https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi/simple --install-preview true
```

Comment:
Nit. If we go multiline, I prefer to commit entirely to the multiline. Check the commands under `sphinx` for examples of what I mean. 

```
{envbindir}/python {toxinidir}/../../../eng/tox/create_wheel_and_install.py \
   -d {distdir} \
   -p {toxinidir} \
   -e https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi/simple \
   --install-preview true
```